### PR TITLE
Use app repo for value service

### PR DESCRIPTION
**Pull request by tomahawk28**
Using `app` repo for value service in all resources. 

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by tomahawk28** [comment]
Tested in `ginger`

1. Deploying this to ginger's `app-operator-unique`.
2. Deleted helm release `chart-operator` several times. 
3. app-operator re-create chart-operator in the next reconciliation.

File: service/controller/app/resource/secret/desired.go [hunk 1]
```diff
@@ -64,8 +65,15 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
...
 
+	bytes, err := yaml.Marshal(mergedData)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
 	secret := &corev1.Secret{
-		Data: mergedData,
+		Data: map[string][]byte{
+			"values": bytes,
```
Comment (tomahawk28) [inline service/controller/app/resource/secret/desired.go:75]: we need an additional top layer called `values` to keep up with older version.

**Review by rossf7 — APPROVED** [review]
Screen Shot 2020-12-03 at 7 03 37 PM

Sweet! 🚀 LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Using Value service from app repository.
```
[the remark this answers is not in the document]
Comment (tomahawk28) [reply CHANGELOG.md:12]: Thanks, I would add it!

**Pull request merged**